Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs

```diff
@@ -31,7 +30,6 @@
 namespace Remotion.Web.Development.WebTesting.IntegrationTests
 {
   [TestFixture]
-  [RequiresUserInterface]
   public class MouseTest : IntegrationTest
```

Comment:
Since we added this in the other project we probably want to remove it from this project?